src/coreclr/scripts/superpmi_setup.py

```diff
@@ -461,10 +461,16 @@ def main(main_args):
             with tempfile.TemporaryDirectory() as jitutils_directory:
                 run_command(
                     ["git", "clone", "--quiet", "--depth", "1", "https://github.com/dotnet/jitutils", jitutils_directory])
+
+                # Make sure ".dotnet" directory exists, by running the script at least once
+                dotnet_script_name = "dotnet.cmd" if is_windows else "dotnet.sh"
+                dotnet_script_path = path.join(source_directory, dotnet_script_name)
+                run_command([dotnet_script_path, "--info"])
+
                 # Set dotnet path to run bootstrap
                 os.environ["PATH"] = path.join(source_directory, ".dotnet") + os.pathsep + os.environ["PATH"]
                 bootstrap_file = "bootstrap.cmd" if is_windows else "bootstrap.sh"
-                run_command([path.join(jitutils_directory, bootstrap_file)], jitutils_directory)
+                run_command([path.join(jitutils_directory, bootstrap_file)])
```

Comment:
I now realized that the error could be because we are not running `bootstrap` inside `jitutils_directory`. I deleted `_cwd` from `run_command`, but I guess it should be there and we should continue passing `jitutils_directory` as current working directory.

Reply:
oh; I deleted this arg because it was "extra", but maybe the cwd should be there? But I still think we should invoke the shell script first to ensure the .dotnet directory is there.

Reply:
Yes, cwd should be there so we can run it inside `jitutils_directory`. That should fix the problem.

And yes, we can run `dotnet --info` just to be sure.